### PR TITLE
Add revision of CRISPR screen

### PR DESCRIPTION
**Pull request by miltondp**
Add changes by Sumei to respond to reviewer's comments.

### CODE REVIEW

File: content/07.00.methods.md [hunk 1]
```diff
@@ -341,9 +349,11 @@ Cell images were captured through fluorescent microscope EVOS for GFP signal det
 **Fluorescence-activated cell sorting (FACS).**
 Cells were immediately collected into 50ml tubes (From this point on, keep cells cold), and spun at 500 x g for 5min at 4oC.
 After DPBS wash, cell pellets were resuspended with FACS Sorting Buffer (1x DPBS without Ca2+/Mg2+, 2.5mM EDTA, 25mM HEPES, 1% BSA.
-The solution was filter sterilized, and kept at 4oC), pi-pet gently to make single cells.
+The solution was filter sterilized, and kept at 4oC), (pi-pet gently) with gentle pipetting to make single cells.
```
Comment (cgreene) [inline content/07.00.methods.md:352]: can't tell what's happening here with the parentheses

**Review by cgreene — COMMENTED** [review]
one spot i don't understand

File: content/07.00.methods.md (hunk 1 above)
Comment (miltondp) [reply content/07.00.methods.md:352]: Me neither. I just copied/pasted what Sumei wrote. Maybe we can leave the "with gentle pipetting" without the part in parenthesis?

Comment (miltondp) [reply content/07.00.methods.md:352]:
```suggestion
The solution was filter sterilized, and kept at 4oC), with gentle pipetting to make single cells.
```

**Comment by cgreene** [comment]
I agree

**Comment by miltondp** [comment]
Ok. I'll merge the PR then.

**Pull request merged**